thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1447,6 +1447,7 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
     IBitSet *queryRhsChannelStopSet() { dbgassertex(0 == queryJobChannelNumber()); return rhsChannelStop; }
     void startLeftInput()
     {
+        LookAheadTimer t(slaveTimerStats, timeActivities);
```

Comment:
why not around the startInput(0) ?

Reply:
The changes related to lookahead in thlookupslave is being done in https://github.com/hpcc-systems/HPCC-Platform/pull/19261.  This change is removed from this PR.